src/simplewallet/simplewallet.cpp

```diff
@@ -8826,45 +8842,61 @@ bool simple_wallet::churn(const std::vector<std::string> &args_)
   int churn_number, pause_time, i;
   srand (time(NULL)); // seed RNG with current time
   churn_number = rand()%(8 + 1 - 2) + 2; // churn between 2 to 8 times
-  pause_time = rand()%(60 + 1 - 30) + 30; // wait between 30 minutes to 1 hour per each churn
+  pause_time = rand()%(120 + 1 - 60) + 60; // wait between 1  to 2 hours per interval
   float total_time = static_cast<float>(churn_number*pause_time)/60; // calculate how long churning will take in hours
-  printf ("\nTo mitigate the risk of spend time analysis, the number of churns and interval pause time are selected at random.\n");
-  printf ("Now churning %d times with %d-minute intervals, keep your wallet open for %.2f hours.\n", churn_number, pause_time, total_time);
+
+  // Calculate fee
+  float churn_fee = churn_number * 0.032654; // usual 2/2 tx fee
+  if (m_wallet->use_fork_rules(HF_VERSION_PER_BYTE_FEE))
+  {
+    float churn_fee = churn_number * 0.015;
```

Comment:
Here, float will mask the var in the outside scope, which is probably not wanted.